### DISCUSSION TITLE
Coloring for address overlay

### DISCUSSION THREAD

**Discussion by tommycrock** [streetcomplete/StreetComplete#5422] [Ideas]
I think it would be helpful for the address overlay to feature colours for buildings, just like ways are colored for surfaces, pavement, etc. Currently, if I switch to addresses layer and am zoomed out to 17 or below (I think) I can't see if the addresses are mapped making it harder to focus my efforts (which way should I walk). Single missing addresses are hard to see. Additionally, newly split/mapped buildings are not visible until you click on them, nor is it apparent if addresses are without street/place. 

Is this possible? In quest mode nearby buildings are highlighted in purple so it seems like it should be. If so, I think the only question would be the colour scheme. Red is typically missing but should probably be reserved for buildings of the types that trigger the address quest and entrances. Green for housenumber and street/place. Then different colours for buildings or entrances with no street/place (yellow?), only housename (no number - sometimes okay, sometimes missing (blue?)), no housenumber/name but street/place (orange?), other building types without address (grey/black?).

What do you think?

**Comment by arrival-spring** [comment]
I think this makes some sense, however it is complicated by the different practices in different countries of address on building or on node on outline or within outline. Definitely house number but missing street/place could be highlighted I'd think.

**Comment by mnalis** [comment]
- ref: https://github.com/streetcomplete/StreetComplete/issues/5748
